### PR TITLE
ISPN-6927 Remove acquireRemoteLock and gtx from ClusteredGetCommand

### PR DESCRIPTION
**Pull request by rvansa**
https://issues.jboss.org/browse/ISPN-6927

### CODE REVIEW

File: core/src/main/java/org/infinispan/commands/CommandsFactory.java [hunk 1]
```diff
@@ -289,7 +289,13 @@
     * @param flagsBitSet Command flags provided by cache
     * @return a ClusteredGetCommand
     */
-   ClusteredGetCommand buildClusteredGetCommand(Object key, long flagsBitSet, boolean acquireRemoteLock, GlobalTransaction gtx);
+   ClusteredGetCommand buildClusteredGetCommand(Object key, long flagsBitSet);
+
+   @Deprecated
+   default ClusteredGetCommand buildClusteredGetCommand(Object key, long flagsBitSet, boolean acquireRemoteLock, GlobalTransaction gtx) {
+      if (acquireRemoteLock) throw new UnsupportedOperationException();
```
Comment (slaskawi) [inline core/src/main/java/org/infinispan/commands/CommandsFactory.java]: Could you please add some information to the exception?

**Comment by rvansa** [comment]
Rebased and added exception message.

**Comment by slaskawi** [comment]
LGTM

**Comment by slaskawi** [comment]
Integrated thanks!

File: core/src/main/java/org/infinispan/persistence/cluster/ClusterLoader.java [hunk 2]
```diff
@@ -60,8 +61,8 @@ public MarshalledEntry load(Object key) throws PersistenceException {
       if (!isCacheReady()) return null;
 
       ClusteredGetCommand clusteredGetCommand = new ClusteredGetCommand(
-            key, cacheName, EnumUtil.EMPTY_BIT_SET, false, null,
-            cache.getCacheConfiguration().dataContainer().keyEquivalence());
+            key, cacheName, EnumUtil.bitSetOf(Flag.SKIP_OWNERSHIP_CHECK),
```
Comment (danberindei) [inline core/src/main/java/org/infinispan/persistence/cluster/ClusterLoader.java]: Was this intentional?

Comment (rvansa) [inline core/src/main/java/org/infinispan/persistence/cluster/ClusterLoader.java]: Seems like a change that got in from the scattered cache (there I use `SKIP_OWNERSHIP_CHECK` more extensively as it has better meaning than just `CACHE_MODE_LOCAL`). It is unintentional within a scope of this PR.

**Pull request closed without merging**